linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_login_banner_text/tests/missing_value_stig.fail.sh

```diff
@@ -10,7 +10,7 @@ install_dconf_and_gdm_if_needed
 # expanded=$(echo "$login_banner_text" | sed 's/(\\\\\x27)\*/\\\x27/g;s/(\\\x27)\*//g;s/(\\\\\x27)/tamere/g;s/(\^\(.*\)\$|.*$/\1/g;s/\[\\s\\n\][+*]/ /g;s/\\//g;s/(n)\*/\\n/g;s/\x27/\\\x27/g;')
 
 clean_dconf_settings
-add_dconf_setting "org/gnome/login-screen" "banner-message-enabled" "true" "gdm.d" "00-security-settings"
-add_dconf_lock "org/gnome/login-screen" "banner-message-enabled" "gdm.d" "00-security-settings-lock"
+add_dconf_setting "org/gnome/login-screen" "banner-message-enabled" "true" "distro.d" "00-security-settings"
+add_dconf_lock "org/gnome/login-screen" "banner-message-enabled" "distro.d" "00-security-settings-lock"
```

Comment:
Same comments here about applicability, either this test is moved to RHEL9 platform, or the test remains setting up `gdm.d`.